vignettes/wwinference.Rmd

```diff
@@ -353,7 +367,21 @@ state-level R(t) estimate.
 We can generate this directly on the output of `wwinference()` using:
 ```{r extracting-draws}
 draws_df <- get_draws_df(ww_fit)
+
+# Print the names of the variables included in the output dataframe
+cat(
+  "Variables in dataframe: ",
+  sprintf("%s", paste(unique(draws_df$name), collapse = ", "))
+)
 ```
+Note that by default the `get_draws_df()` function will return  a tidy long
+dataframe with all of the posterior draws joined to applicable data for each of
+the included variables. If only interested in the hospital admissions estimates
+(`"predicted counts"`) or the wastewater estimates (`"predicted wastewater"`),
+we suggest immediately filtering to `name %in% c(your_desired_outputs)`
+so as not to carry around extra data.
+This is particularly important if there are a large
+number of wastewater sites, as this dataframe can get quite large.
```

Comment:
I would strike this part. I think the user can work it out. If we really want to encourage users to filter at this step, `get_draws_df` should take an optional argument specifying which outputs to return.
```suggestion
the included variables. 
```

Reply:
Yeah so that was my idea with the `included_variables` thing. I can work on it in a separate PR though, I got super stuck even though it should be quite simple.

My push back here, for now at least until we have that feature, is that we specifically got user feedback saying that it was not at all clear that there were multiple output variables/types in the `draws_df`. So while verbose to write all of this, I think it is important that we have something, to prevent the mistake of treating this as just a dataframe of fitted and forecasted hospital admissions (which if you just do `head(draws_df)` thats what it looks like). I

Reply:
Made an alternative suggestion.